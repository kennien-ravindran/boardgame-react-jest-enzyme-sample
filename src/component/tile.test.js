import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Tile from './tile';


Enzyme.configure({ adapter: new Adapter() })

describe('Tile', () => {
  it('should render correctly', () => {
    const output = shallow(
      <Tile row={0} column={0} value="X" />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
