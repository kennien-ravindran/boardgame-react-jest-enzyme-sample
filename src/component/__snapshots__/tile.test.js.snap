// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile should render correctly 1`] = `
<div
  className="tile row0 column0"
  id="0x0"
  onClick={[Function]}
>
  X
</div>
`;
